package.json

```diff
@@ -30,7 +30,6 @@
         "api-catalog-service-debug": "java $npm_package_config_jvmArgs -Xdebug -Xrunjdwp:transport=dt_socket,server=y,address=5014,suspend=n -jar api-catalog-services/build/libs/api-catalog-services.jar --spring.config.additional-location=file:./config/local/api-catalog-service.yml",
         "cloud-gateway": "java $npm_package_config_jvmArgs -javaagent:scripts/jacocoagent.jar=includes=org.zowe.*,output=tcpserver,address=*,port=6301 -jar cloud-gateway-service/build/libs/cloud-gateway-service.jar --spring.config.additional-location=file:./config/local/cloud-gateway-service.yml --apiml.security.ssl.verifySslCertificatesOfServices=true",
         "//1": "For api-catalog-service-standalone, api-catalog-ui needs to be built for standalone mode. See README in catalog-ui for further details",
-        "api-catalog-service-standalone": "REACT_APP_API_PORTAL=true && java $npm_package_config_jvmArgs -jar api-catalog-services/build/libs/api-catalog-services.jar --spring.profiles.active=standalone --spring.config.additional-location=file:./config/local/api-catalog-service.yml --apiml.security.ssl.verifySslCertificatesOfServices=true",
```

Comment:
I'd leave this. It's still valid, just remove `REACT_APP_API_PORTAL=true` from the command